### ISSUE TITLE
Дефолтный шаблон.

### ISSUE THREAD

**Issue by Sergeysem** [maxsite/cms#274]
Хотел спросить, как можно вылечить дерганье дефолтного шаблона. Например обновляем страницу или кликаем по меню. 
Для примера вот этот сайт.. http://friendly-urls.vizr.ru

**Comment by maxsite** [comment]
Что-то такого не наблюдаю.

**Comment by Sergeysem** [comment]
Не всегда, из 10 обновлений страницы, 5 точно дергается. Пробовал в вивалди, хроме, опере.
Могу видео записать.

**Comment by Sergeysem** [comment]
Видео.
https://drive.google.com/file/d/1358hi0Qme2wdhZ6Nuq09KCNpaHhBGtYR/view?usp=sharing

**Comment by Sergeysem** [comment]
Добавил в css в html и body строчку overflow:scroll; 
Вроде получше стало, но надписи меню все равно "ездят"

**Comment by Sergeysem** [comment]
http://max-3000.com здесь нет такого, все стоит как вкопанное. Я так понимаю шаблон сайта сделан на основе дефолтного.

**Comment by maxsite** [comment]
Наверное такой html-код. Может плагин какой-то сбивает разметку.

**Comment by Sergeysem** [comment]
Нет, установка с нуля, чистый движок и шаблон, все дергается уже на этом этапе.
Посмотрел, на 93 версии вроде дефолтный по стабильней себя ведет, а уже с 96 вроде, начинает плясать.

**Comment by Sergeysem** [comment]
Хотя соврал, с 93 по 99 стабильные при условии добавления строки в css overflow:scroll;
Но 100 шаблон, это совсем другой шаблон. Поправить бы.

**Comment by maxsite** [comment]
Не вижу особой проблемы. После загрузки все корректно отображается.

**Comment by Sergeysem** [comment]
Да знать бы где поправить, в каких файлах, сам бы все сделал, а так, это не реально найти.
То что на место встает, это понятно, но сам факт дерганья, это уже не нормально и не приятно для глаз.

**Comment by Sergeysem** [comment]
Готов согласиться, но тут http://max-3000.com все хорошо, даже отлично. Ни куда ни что не убегает, буквы не дергается, меню не дергается, шаблон как вкопанный.
